host/types/types.go

```diff
@@ -214,6 +214,7 @@ type ActiveJob struct {
 	Job        *Job      `json:"job,omitempty"`
 	HostID     string    `json:"host_id,omitempty"`
 	InternalIP string    `json:"internal_ip,omitempty"`
+	PID        int       `json:"pid,omitempty"`
```

Comment:
What do you think about making this a pointer like `ExitStatus`? Then it is set iff the job is running.